diagnostic.md

```diff
@@ -94,4 +95,4 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+exercise, given surname,  jason weeks.
```

Comment:
You got the first and third ones correct.  The value of teamMember['name'] is the entire nested object, { given: 'Jason', surname: 'Weeks' }, not just the keys (given and surname).